otel-integration/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## OpenTelemtry-Integration
 
+### v0.0.150 / 2025-02-29
+
+- [Feat] Upgrade OpenTelemetry Collector to `0.120.0`
```

Comment:
No breaking change notice required, right? Maybe we could put that note regarding underscore and dots here?